docs/source/changelog.rst

```diff
@@ -1,4 +1,52 @@
+*********
 Changelog
-========================
+*********
 
-the newest updates
\ No newline at end of file
+manimce-v1.0.0-dev
+==================
+
+:Date: N/A
+
+New Features
+-------------
+#. Add 4k rendering option
+#. Added clearer installation instructions and quick-use scripts
+#. Added tests to ensure stuff doesn’t break between commits (For developers) [Uses Github CI, and Pytest]
+#. high quality has gotten its own short argument (-e), and the quality argument explanations have been improved
+#. Implement logging with :code:`rich` and :code:`logger` instead of plain ol` prints.
+#. Add customizable left and right bracket for :code:`Matrix` mobject and :code:`set_row_colors` method for matrix mobject
+#. Added a flag :code:`dry_run`, which doesn’t write any media.
+#. Allow for running manim with :code:`python3 -m manim`
+#. Add :code:`AddTeXLetterByLetter` animation
```

Comment:
issue ; this has not been added (#84 is closed )

Reply:
That PR was merged right?

Reply:
wut ?! I'm confused, I thought it was broken and I closed the PR. Maybe not so lol.